src/components/common/Label/styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const StyledLabel = styled.label`
+  font-weight: bold;
+  width: 500px;
+
+  div {
+    color: red;
+    display: inline-block;
+    margin-left: -5px;
```

Comment:
Usually this is a code smell, can this be done any other way?

Reply:
I set the left margin in this way to try and close the space between by using `display: inline-block`. I was able to get around this by setting `display: flex` in the parent div